### ISSUE TITLE
Please correct Definition of Property "Bounding Box"

### ISSUE THREAD

**Issue by leightonlc** [DOI-DO/dcat-us#200]
_This information will help the Core Team resolve issues with the draft DCAT3-US application profile.  So we can track all issues, please submit different issues in separate issue tickets. The Core Team will review and adjudicate the submitted issue and provide comments on this ticket indicating steps taken to address the issue as submitted._

**Name:**  _Leighton Christiansen_

**Affiliation:** _Centers for Disease Control and Prevention (CDC)_

**Type of issue:** _**Schema** (Property)_

**Issue:** _[one issue per ticket please.... identify and describe the issue, providing a link to the closest section in the document]_
- _https://doi-do.github.io/dcat-us/#location-bbox_
- _The Definition give reads: "The geographic center (centroid) of a spatial thing." This is actually the definition of "Centroid."_

**Recommended change(s):** _Please replace this definition with something along the lines of "an area defined by two longitudes and two latitudes". Thank you! _

**Comment by jbrown-xentity** [comment]
Agreed, this seems like a bad copy/paste. Need resolution on this to avoid confusion, as the translation in spatial will be one of the more complex transitions from DCAT-US1.1 to 3.0...
